code/__HELPERS/unsorted.dm

```diff
@@ -41,6 +41,9 @@
 	else if(dx<0)
 		.+=360
 
+/proc/get_angle(atom/a, atom/b)
```

Comment:
I'm unsure this has the same results as `Get_Pixel_Angle` but anyway give it a different name, it can be easily confused with `Get_Angle`.